src/main/java/seedu/address/model/Model.java

```diff
@@ -70,7 +71,22 @@
     void deleteClient(Client target);
 
     /**
-     * Adds the given client. {@code client} must not already exist in the address book.
+     * Sets the widget box contents to the given content.
+     */
+    void setWidgetContent(Object content);
```

Comment:
Should `setWidgetContent` be in `Model`? Seems like this is mixing UI-related stuff with model-related stuff.